File: healthone/lib/exercise_meditation_folder/pass_meditation.dart
class PassMeditation {
  final int passmeditationid;
  final String meditationname;
  final String username;
  final String dateof;
  final int totaltime;
  final String soundused;

  PassMeditation({
    required this.passmeditationid,
    required this.meditationname,
    required this.username,
    required this.dateof,
    required this.totaltime,
    required this.soundused,
  });

  factory PassMeditation.fromMap(Map studentMeditatingMap){
    return PassMeditation(
      passmeditationid: studentMeditatingMap['passmeditationid'],
      meditationname: studentMeditatingMap['meditationname'],
      username: studentMeditatingMap['username'],
      dateof: studentMeditatingMap['dateof'],
      totaltime: studentMeditatingMap['totaltime'],
      soundused: studentMeditatingMap['soundused'],
    );
  }

}//class

File: mentalhealth/src/main/java/com/example/mentalhealth/exercisemental/PassMeditation.java
package com.example.mentalhealth.exercisemental;
import jdk.jfr.DataAmount;
//import lombok.RequiredArgsConstructor;
import javax.persistence.*;
import java.util.Set;
import lombok.Data;
import lombok.RequiredArgsConstructor;
import java.time.LocalDateTime;

@Data
@Entity
@Table(name = "passmeditation")
@RequiredArgsConstructor
public class PassMeditation {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "passmeditationid")
    private int passmeditationid;
    //@Column(name = "exercisename", nullable=false)
    private String meditationname = "test";
    //@Column(name = "username", nullable=false)
    private String username = "test";
    //@Column(name = "startedat", nullable=false)
    private String dateof = "test";
    //@Column(name = "totaltime", nullable=false)
    private int totaltime;
    //@Column(name = "caloriesburned", nullable=false)
    private String soundused = "test";

    public String getMeditationname(){
        return meditationname;
    }

    public String getUsername(){
        return username;
    }

    public String getDateof(){
        return dateof;
    }

    public int getTotaltime(){
        return totaltime;
    }

    public String getSoundused(){
        return soundused;
    }

}